scss/_patterns_list-tree.scss

```diff
@@ -46,26 +42,32 @@
     @include vf-focus;
 
     margin-top: $sp-xx-small;
-    padding-left: 0.8rem;
+    padding-left: map-get($grid-margin-widths, small);
```

Comment:
I managed to unify this by moving all the padding to the items themselves, rather than the parent. It's important that the items themselves have padding so that the items with toggle icons line up horizontally with the ones that don't.